vmdb/app/views/layouts/_global_header.html.haml

```diff
@@ -0,0 +1,70 @@
+!!! XML
+!!! Strict
+%html
+  %head
+    %meta{"http-equiv" => "Pragma", :content => "no-cache"}
+    %meta{"http-equiv" => "Pragma", :content => "no-store"}
+    %meta{"http-equiv" => "Pragma", :content => "must-revalidate"}
+    %meta{"http-equiv" => "cache-control", :content => "no-cache, no-store"}
+
+    %title
+      = h(title_from_layout(@layout))
+
+    = stylesheet_link_tag 'application'
+    = stylesheet_link_tag 'main'
+    = tag :link, :rel => "shortcut icon", :href => image_path("favicon.ico")
+    = stylesheet_link_tag 'glyphicons'
+    = stylesheet_link_tag 'template'
+    = render :partial=>"stylesheets/template50"
+    <!--[if IE 8]>
+    <html class="ie8"><![endif]-->
+    = javascript_include_tag 'jquery/jquery-1.8.3.min','jquery/jquery-ui-1.9.2.custom.min'
+    %script{:type => "text/javascript"}
+      $j = jQuery;
+
+    = stylesheet_link_tag "jquery/miq/jquery-ui-1.9.2.custom"
+
+    = javascript_include_tag 'jquery.observe_field'
+    = javascript_include_tag 'miq_jquery_ujs_rails3'
+    = javascript_include_tag 'miq_ujs_bindings'
+    = javascript_include_tag 'jquery.form'
+    = javascript_include_tag 'jquery.cookie'
+
+    = csrf_meta_tag
+
+    = javascript_include_tag 'prototype_min', 'effects_min', 'application', 'dragdrop_min', 'controls_min'
+
+    = javascript_include_tag 'dhtmlx_gpl_36/dhtmlxcommon.js', 'dhtmlx_gpl_36/dhtmlxcontainer.js'
+
+    = stylesheet_link_tag '/assets/dynatree_1.2.4/skin-cfme/ui.dynatree.css'
+    = javascript_include_tag 'dynatree_1.2.4/jquery.dynatree', 'cfme_dynatree'
+
+    = stylesheet_link_tag 'slick.grid.css', 'cfme_slickgrid'
+    = javascript_include_tag 'SlickGrid-2.1/jquery.event.drag-2.2', 'SlickGrid-2.1/slick.core',
+      'SlickGrid-2.1/slick.grid', 'SlickGrid-2.1/slick.dataview', 'SlickGrid-2.1/plugins/slick.autotooltips',
+      'cfme_slickgrid'
+
+    = javascript_include_tag 'codemirror-4.2/lib/codemirror.js', 'codemirror-4.2/mode/yaml/yaml.js'
+    = stylesheet_link_tag 'codemirror-4.2/lib/codemirror.css'
+
+    - if @timeline
+      = javascript_include_tag '/javascripts/timeline/api/timeline-api.js'
+      - if @timeline_filter
+        = javascript_include_tag '/javascripts/timeline/api/scripts/miq_filters.js'
+        = javascript_include_tag '/javascripts/timeline/api/scripts/miq_layouts.js'
+        = javascript_include_tag '/javascripts/timeline/api/scripts/miq_painters.js'
+
+    = javascript_include_tag 'spin'
+
+    %style html, body{width:100%; height:100%; margin:0px; padding:0px; overflow:hidden;}
+
+  - if ["configuration","policy"].include?(@layout)
+    - if is_browser_ie?
+      %body{:id => @config_tab, :onresize => "miqOnResize();"}
+    - else
+      %body{:onload => "miqOnLoad();", :id => @config_tab, :onresize => "miqOnResize();"}
+  - else
+    - if is_browser_ie?
+      %body{:id => @layout, :onresize => "miqOnResize();"}
+    - else
+      %body{:onload => "miqOnLoad();", :id => @layout, :onresize => "miqOnResize();"}
```

Comment:
please refactor this to have
`an_id = ["configuration","policy"].include?(@layout) ? @config_tab : @layout]`
and then just one code path
thx!